db/migrate/20191217172047_change_name_of_spaces_column_to_space_in_requests_table.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeNameOfSpacesColumnToSpaceInRequestsTable < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :requests, :spaces_id, :space_id
+  end
+end
```

Comment:
I see now that you have revised the table column name here. Should it be better if we edit the previous file and delete this one? Because it does not explain anything regarding the project flow, as it is just a typo.